network/requestTracker.go

```diff
@@ -285,8 +285,11 @@ func (rt *RequestTracker) Accept() (conn net.Conn, err error) {
 		rt.hostRequests.pruneRequests(rateLimitingWindowStartTime)
 		originConnections := rt.hostRequests.countOriginConnections(trackerRequest.remoteHost, rateLimitingWindowStartTime)
 
+		remoteHostIsNonLocal := (!rt.config.DisableLocalhostConnectionRateLimit) || (!isLocalhost(trackerRequest.remoteHost))
```

Comment:
I prefer naming this rateLimitedRemoteHost
This can be local host but reads remote Host Is Non Local.